krun.py

```diff
@@ -232,7 +229,7 @@ def run(self):
 
         info("Done: Results dumped to %s" % self.out_file)
         if errors:
-            warn("%s ERRORS OCCURRED! READ THE LOG!%s" % (ANSI_RED, ANSI_RESET))
+            warn("Errors occurred! Read the scrollback!")
```

Comment:
We've not discussed that, but I don't object. In fact I think it's a good idea.

Let's dump whatever goes via the logging API to a date-stamped file and mail errors as they occur? The latter should be controlled by a config option. Sounds good?